.github/workflows/readme.yml

```diff
@@ -27,5 +27,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install invoke rundoc .
           python -m pip install tomli
+          python -m pip install slack-sdk
+          python -m pip install openpyxl
```

Comment:
why are these needed for the readme? I don't think anything in the readme should be posting to slack or creating excel files